src/theme/base/_variables.scss

```diff
@@ -1,4 +1,13 @@
 $font-main: 'Open Sans', sans-serif;
-
+// Palette
 $color-main: #EB6E64;
 $color-white: #FFF;
+// Font-sizes
+$font-xxs: 0.75rem; // 12px
+$font-xs: 0.875rem; // 14px
+$font-s: 1rem; // 16px
+$font-m: 1.125rem; // 18px
+$font-l: 1.5rem; // 24px
+$font-xl: 2rem; // 32px
+$font-xxl: 4.25rem; // 68px
+$font-giant: 4.25rem; // 84px
```

Comment:
I think you need to update the size here like:

```diff
- $font-giant: 4.25rem; // 84px
+ $font-giant: 5.25rem; // 84px
```

Reply:
duude.. really! Happened the same in the old PR! Good eye mate!

Good job